iroha/src/wsv.rs

```diff
@@ -18,6 +18,8 @@ pub struct WorldStateView {
     pub transactions: HashMap<Hash, TransactionValue>,
```

Comment:
We no longer need a map here now. It can be as it was previously simply a hash set of tx hashes. As all transactions for queries can be taken from blocks.

Reply:
I know, that it doubles space, but it gives better time access to Transactions.

Reply:
Given the double size and blockchains are sometimes hundreds of gigabytes, I think it is better to have a slighlty longer query,

Reply:
Also I don't see how the query will be slower - we simply iterate blocks and their txs filtering them by account and dates. Same O(n) as for simply going through a map iter of txs.